pkg/utils/webhook/certificates.go

```diff
@@ -117,8 +169,8 @@ func GenerateNewCAAndServerCert(namespace, name string, caConfig certificates.Ce
 	)
 
 	serverConfig := &certificates.CertificateSecretConfig{
-		CommonName:  name,
```

Comment:
we should keep that name as that function could be or is used to generally generate a ca server cert

Reply:
@schrodit The `CommonName` was previously derived from the service name. When we just have the webhook URL, how should it be set?

Reply:
I would have the value as another function parameter. With that we can configure it to use `landscaper-webhhok` but others have the chance to use whatever they want

Reply:
Ok, understood.